<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fl文件整合</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>项目：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="project_cb">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>场次：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="scene_cb">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>镜头：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="shot_cb">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>环节：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="target_cb">
       <item>
        <property name="text">
         <string>Fl</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="import_abc_pb">
       <property name="text">
        <string>导入ABC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="import_fur_pb">
       <property name="text">
        <string>导入Fur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_as_pb">
       <property name="text">
        <string>另存文件</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
